pkg/distributor/ha_tracker.go

```diff
@@ -172,6 +174,10 @@ func (c *haTracker) checkKVStore(ctx context.Context, key, replica string, now t
 	})
 }
 
+func replicasNotMatchError(replica, elected string) error {
+	return httpgrpc.Errorf(http.StatusAccepted, "replicas did not mach, rejecting sample: replica=%s, elected=%s", replica, elected)
+}
```

Comment:
Spurious change?

Reply:
This appeared after I rebased amend a commit, I wanted to trick the CI to trigger a build as it was failing on a flaky test on the ruler.

So this is from master

Reply:
I rebased it again and its gone !